lib/region/europe/countryCodes.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file List of country codes legally considered part of the EU region.
+ */
+
+var countryCodes = {
```

Comment:
Thanks. I see it as a matter of personal taste, since you're right that for this module there are practically zero performance differences either way. Your reasoning is totally sound.